lib/src/firebase_performance_dio.dart

```diff
@@ -57,9 +61,10 @@ class DioFirebasePerformanceInterceptor extends Interceptor {
   }
 
   @override
-  Future onError(DioError err, ErrorInterceptorHandler handler) async {
+  Future onError(DioException err, ErrorInterceptorHandler handler) async {
     try {
-      final requestKey = err.requestOptions.extra.hashCode;
+      final requestKey = generateKey?.call(err.requestOptions.extra) ??
+          err.requestOptions.extra.toString().hashCode;
```

Comment:
Why we need to use the toString() method ?

Reply:
Got it. In this case I believe we can use the `toString().hashCode` for each `requestKey`